opam

```diff
@@ -1,4 +1,6 @@
-opam-version: "1"
+opam-version: "1.2"
+name: "sodium"
+version: "dev"
```

Comment:
can you please not? "dev" versions are annoying to use locally, it's much better to put the next-to-be-released version into opam.